src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -3,103 +3,309 @@
+    type: command
+    short-summary: Download and install the DC/OS or Kubernetes command-line tool for a cluster.
+"""
+
 helps['acs kubernetes'] = """
     type: group
-    short-summary: Manage a Kubernetes orchestrated Azure Container Service.
+    short-summary: Commands to manage a Kubernetes-orchestrated Azure container service.
 """
+
 helps['acs kubernetes get-credentials'] = """
     type: command
     short-summary: Download and install credentials to access a cluster.
 """
+
+helps['acs list-locations'] = """
+    type: command
+    short-summary: List locations where Azure container service is in preview and in production.
+"""
+
 helps['acs scale'] = """
     type: command
     short-summary: Change the private agent count of a container service.
+    parameters:
+        - name: --new-agent-count
+          type: int
+          short-summary: The number of agents for the container service.
 """
-helps['acs install-cli'] = """
+
+helps['acs show'] = """
     type: command
-    short-summary: Download and install the DC/OS and Kubernetes command line for a cluster.
+    short-summary: Show details of a container service.
 """
-helps['acs create'] = """
+
+helps['acs wait'] = """
     type: command
-    short-summary: Create a new Azure Container Service cluster.
-    examples:
-        - name: Create a default acs cluster.
-          text: az acs create -g MyResourceGroup -n MyContainerService
-        - name: Create a Kubernetes cluster.
-          text: az acs create --orchestrator-type Kubernetes -g MyResourceGroup -n MyContainerService
-        - name: Create a Kubernetes cluster with a an existing SSH key.
-          text: az acs create --orchestrator-type Kubernetes -g MyResourceGroup -n MyContainerService --ssh-key-value /path/to/publickey
-        - name: Create an ACS cluster with two agent pools.
-          text: az acs create -g MyResourceGroup -n MyContainerService --agent-profiles "[{'name':'agentpool1'},{'name':'agentpool2'}]"
-        - name: Create an ACS cluster where the second agent pool has a vmSize specified.
-          text: az acs create -g MyResourceGroup -n MyContainerService --agent-profiles "[{'name':'agentpool1'},{'name':'agentpool2','vmSize':'Standard_D2'}]"
-        - name: Create an ACS cluster with agent-profiles specified from a file.
-          text: az acs create -g MyResourceGroup -n MyContainerService --agent-profiles MyAgentProfiles.json
+    short-summary: Wait for a container service to reach a desired state.
 """
+
+# AKS command help
+
 helps['aks'] = """
      type: group
-     short-summary: Manage Kubernetes clusters.
+     short-summary: Manage Azure Kubernetes clusters.
 """
+
+helps['aks browse'] = """
+    type: command
+    short-summary: Open a web browser to the dashboard for a managed Kubernetes cluster.
+    parameters:
+        - name: --disable-browser
+          type: bool
+          short-summary: Don't launch a web browser after establishing port-forwarding.
```

Comment:
It sets up local port-forwarding to the remote Kubernetes dashboard using the `kubectl` CLI. It prints out the details of the port-forwarding and then launches the browser--or doesn't in this case.
I suppose this command could still be useful without launching the browser in some case where just the local URL is needed--maybe test automation? Although this command doesn't return until the user stops the port-forwarding with `Ctrl+C`, so that makes automating it difficult...

Reply:
Also, I implemented this command, and copied flags from the existing `az acs browse` command.